components/kyma-environment-broker/internal/ias/bundle_test.go

```diff
@@ -50,7 +50,7 @@ func TestServiceProviderBundle_ConfigureServiceProviderType(t *testing.T) {
 
 	// then
 	assert.NoError(t, err)
-	provider, err := client.GetServiceProvider(FakeProviderID)
+	provider, err := client.GetServiceProvider(FakeGrafanaID)
```

Comment:
missing test for `ConfigureServiceProviderType` method for `saml2` type

Reply:
Added.